app/simnet_test.go

```diff
@@ -58,11 +58,21 @@ func TestSimnetNoNetwork_TekuVC(t *testing.T) {
 	testSimnet(t, args, false)
 }
 
-func TestSimnetNoNetwork_WithOnlyProposerMockVCs(t *testing.T) {
+func TestSimnetNoNetwork_WithProposerMockVCs(t *testing.T) {
 	testSimnet(t, newSimnetArgs(t), true)
 }
 
-func TestSimnetNoNetwork_MockVCs(t *testing.T) {
+func TestMultipleProposers(t *testing.T) {
```

Comment:
cool will take a look asap

Reply:
ok, found the problem. The test shuts down to early because it includes `DutyRandao` in the results. So if it gets `DutyRandao` and `DutyProposer` result is stops, but then `DutyProposer` is still ongoing, so shutdown blocks it from completing and things fail.

Suggest:
```
BroadcastCallback: func(ctx context.Context, duty core.Duty, key core.PubKey, data core.AggSignedData) error {
  if duty.Type == core.DutyRandao {
    // Ignore duty randao in results since it is internal
    return nil
  }					
  results <- simResult{Duty: duty, Pubkey: key, Data: data}
  return nil
},
```